doc/source/contributing.rst

```diff
@@ -287,6 +286,45 @@ is usually quite straightforward.
 ------------------------------
 
 Once you've made changes and pushed them to your forked repository, you then
-submit a pull request to have them integrated into the *geopandas* code base.
+submit a pull request to have them integrated into the *GeoPandas* code base.
 
 You can find a pull request (or PR) tutorial in the `GitHub's Help Docs <https://help.github.com/articles/using-pull-requests/>`_.
+
+Style Guide & Linting
+---------------------
+
+- GeoPandas supports Python 2 (2.7+) and Python 3 (3.5+) with a single
+  code base. Use modern Python idioms when possible that are
+  compatible with both major versions, and use the
+  [six](https://pythonhosted.org/six) library where helpful to smooth
+  over the differences. Use `from __future__ import` statements where
+  appropriate. Test code locally in both Python 2 and Python 3 when
+  possible (all supported versions will be automatically tested on
+  Travis CI).
```

Comment:
I will remove this part here again, as it is already at the top of the file, I see now